packages/api/lambdas/publish-granules.js

```diff
@@ -14,8 +14,14 @@ const handler = async (event) => {
 
   const promisedPublishEvents = event.Records.map(
     (record) => {
-      const granule = attr.unwrap(record.dynamodb.NewImage);
-      return publishSnsMessage(topicArn, granule);
+      const eventType = record.eventName;
```

Comment:
Can we use `Create/Update/Delete` event to be consistent with `report-collections-topic`?

Reply:
sure